core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackendUtil.scala

```diff
@@ -119,21 +122,25 @@ private[mesos] object MesosSchedulerBackendUtil extends Logging {
   }
 
   /**
-   * Setup a docker containerizer
+   * Setup a docker containerizer from MesosDriverDescription scheduler properties
    */
   def setupContainerBuilderDockerInfo(
     imageName: String,
-    conf: SparkConf,
+    conf: Map[String, String],
     builder: ContainerInfo.Builder): Unit = {
+    val forcePullImage = conf
```

Comment:
Yeah, so as i already mentioned above, `MesosClusterScheduler` uses this method passing a `Map[String, String]` so if we want to keep reusing this method (which makes sense), we will have to go with the Strings.